tests/src/tests.rs

```diff
@@ -489,6 +490,14 @@ fn test(
     ok
 }
 
+fn get_metadata<'a>(line: &'a str, key: &str) -> Option<&'a str> {
+    line.strip_prefix(eco_format!("// {key}: ").as_str())
+}
+
+fn get_flag_metadata(line: &str, key: &str) -> Option<bool> {
+    get_metadata(line, key).map(|value| value == "true")
```

Comment:
Remark to the reviewer: This means that

```
// Ref: whoops
```

equals to 

```
// Ref: false
```

instead of, as before, only the second one bearing any meaning (file wide).

Additionally, for sub-test configuration, both `// Ref: true` and `// Ref: false` behave the same as before, but `// Ref: anything` behaves just like `// Ref: false` instead of being ignored.

I think that's reasonable. But it's different!